apps/pie-docs/src/assets/styles/components/_categorisedIconList.scss

```
@@ -10,15 +11,45 @@ $icon-preview-size: 64px;
     inline-size: $size;
 }
 
+.c-categorisedIconList-search {
+flex-shrink: 0;
+flex-grow: 1;
+}
+
+.c-categorisedIconList-select {
+flex-grow:1;
+
+   @include f.media('>=mid') {
+    flex-grow: 0;
+    }
+
+}
```

The indention is off here. let's make it`s the same as other references. 
```suggestion
.c-categorisedIconList-search {
    flex-shrink: 0;
    flex-grow: 1;
}

.c-categorisedIconList-select {
    flex-grow:1;

    @include f.media('>=mid') {
        flex-grow: 0;
    }
}
```